chainerx_cc/chainerx/routines/manipulation.h

```diff
@@ -11,6 +12,11 @@
 
 namespace chainerx {
 
+enum class PadMode {
+    constant = 1,
```

Comment:
As a convention, please name the members like `kConstant`.

Reply:
Done.